packages/react-positioning/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "@fluentui/react-positioning",
+  "version": "9.0.0-alpha.0",
+  "description": "A react wrapper around Popper.js for Fluent UI",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.1",
+    "@fluentui/scripts": "^1.0.0",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
```

Based on current usage and what I know of patterns moving forward, I think you can get rid of most of these. (We should remove at least react-app-polyfill from the template as well, separately.)
```suggestion
    "@fluentui/eslint-plugin": "^1.0.1",
    "@fluentui/scripts": "^1.0.0",
    "@types/react": "16.9.42",
    "@types/react-dom": "16.9.10",
    "react": "16.8.6",
    "react-dom": "16.8.6"
```